.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "editor.formatOnSave": false
+}
```

Comment:
Please drop this file - IMHO settings files should not be pushed to a repository.

Reply:
Because otherwise my editor wants to reformat everything automatically. Could I add the folder to `.gitignore` instead?

Reply:
Good idea!